core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -28,9 +31,12 @@
 @PublicApi
 @NotThreadSafe
 public final class SetAttributeOptions {
+  private final Logger mLogger = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
```

Comment:
if we have a static logger, testEquals will fail even if we pass in "LOG" in exclude list.
```
java.lang.RuntimeException: java.lang.IllegalAccessException: Can not set static final org.slf4j.Logger field alluxio.client.file.options.SetAttributeOptions.LOG to null value
```

Also tried mock out the static logger here, but still hitting same issue. So just left it as non-static.